docs/commands/ComObject.htm

```diff
@@ -34,9 +34,9 @@ <h2 id="Parameters">Parameters</h2>
 
 <h2 id="Return_Value">Return Value</h2>
 <p>Type: <a href="ComValue.htm#Returns">ComValue</a> or ComObject</p>
-<p>This function returns a COM wrapper object of type dependent on the IID parameter.</p>
+<p>This function returns a COM wrapper object of variant type dependent on the IID parameter.</p>
```

Comment:
If you look to the line above, you will see the *type* is dependent on the IID parameter, not just the variant type. The omission of "variant" was intentional.